unit_test/cryptlib_wrapper/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+    ADD_COMPILE_OPTIONS(-Wno-incompatible-pointer-types -Wno-pointer-sign)
+endif()
+
+INCLUDE_DIRECTORIES(${LIBSPDM_DIR}/include
+                    ${LIBSPDM_DIR}/include/hal
+                    ${LIBSPDM_DIR}/include/hal/${ARCH}
+                    ${LIBSPDM_DIR}/os_stub/include
+                    ${LIBSPDM_DIR}/os_stub
+                    ${LIBSPDM_DIR}/os_stub/cryptlib_mbedtls
+                    ${LIBSPDM_DIR}/os_stub/mbedtlslib/include
+                    ${LIBSPDM_DIR}/os_stub/mbedtlslib/include/mbedtls
+                    ${LIBSPDM_DIR}/os_stub/mbedtlslib/mbedtls/include
+                    ${LIBSPDM_DIR}/os_stub/mbedtlslib/mbedtls/include/mbedtls
+)
+
+add_compile_definitions(LIBSPDM_UNIT_TEST=1)
+
+SET(src_cryptlib_wrapper
+    ${LIBSPDM_DIR}/os_stub/cryptlib_mbedtls/cipher/aead_aes_gcm.c
+    ${LIBSPDM_DIR}/os_stub/cryptlib_mbedtls/cipher/aead_chacha20_poly1305.c
+    ${LIBSPDM_DIR}/os_stub/cryptlib_mbedtls/cipher/aead_sm4_gcm.c
+    hash/sha.c
```

Comment:
`hash/sha.c` is the wrapper and `${LIBSPDM_DIR}/os_stub/cryptlib_mbedtls/hash/sha.c` is the real implementation.